scripts/build/gitlab/configure_gitlab_windows.bat

```diff
@@ -16,7 +16,7 @@ cd ..\..\..\cmake_build
 -DEXTERNAL_SOLVERS_APPLICATION=OFF                                              ^
 -DFLUID_DYNAMICS_APPLICATION=ON                                                 ^
 -DSTRUCTURAL_MECHANICS_APPLICATION=ON                                           ^
--DCONTACT_STRUCTURAL_MECHANICS_APPLICATION=ON                                   ^
+-DCONTACT_STRUCTURAL_MECHANICS_APPLICATION=OFF                                  ^
```

Comment:
Why?, you need to ask first

Reply:
@loumalouomega this is the new CI, not appveyor
currently @Hubertus248 is testing it, and it is not working hence he tries to lower the requirements

I hope I am not wrong @Hubertus248

Reply:
Ok, I have not blocked, just commented

Reply:
After more tinkering build finishes with application enabled, so it's not required anymore